wakamiti-plugins/wakamiti-email/src/main/resources/iti_wakamiti_wakamiti-email.properties

```diff
@@ -4,8 +4,16 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #
 
-mail.server.protocol = the server network protocol {text}
-mail.server.host = the server host {text}
-mail.server.port = the server port {text}
-mail.server.user = the user {text}
-mail.server.password = the password {text}
\ No newline at end of file
+mail.server.protocol=the server's network protocol is {text}.
```

Comment:
los pasos son un poco genericos, hay que pensar que este plugin convivira con otros plugins (por ejemplo el de rest) y que se deberia poder distinguir el paso univocamente.

Se podria cambiar  'the server's network protocol' por 'the email server protocol' , 
'the user is' por 'the email address is',  'the password is' por 'the email password is', 
y asi sucesivamente, para diferenciarlo mas y asegurarnos que no entrara en conflicto con los pasos de otros 
plugins.